pyproject.toml

```
@@ -2,6 +2,99 @@
 # Minimum requirements for the build system to execute.
 requires = ["setuptools", "wheel"]  # PEP 508 specifications.
 
+[tool.setuptools.dynamic]
+readme = {file=["README.rst"]}
+
+[project]
```

This section needs to be removed. Otherwise README modifications done by `setup.py` won't be used. See [this comment] from #806.

```suggestion
[project]
```

[this comment]: /akaihola/darker/issues/806#issuecomment-2629563755